### PR TITLE
(WIP) Kk/instant gas

### PR DESCRIPTION
**Pull request by kkostalkowicz**

### CODE REVIEW

**Comment by maciejka** [comment]
Please check e2e visual regression: https://s3-eu-west-1.amazonaws.com/codechecks-public-artifacts/166230898/60c75485a7981ddacada1d3eeb9c60820d9dcb7e/e2e-vis-reg-report/index.html

**Comment by kkostalkowicz** [comment]
This time there are glitches for iphone6: https://s3-eu-west-1.amazonaws.com/codechecks-public-artifacts/166230898/dd62664832c8e6b5524a25746caa68332e66b911/e2e-vis-reg-report/index.html

Seems an issue for @krzkaczor

**Comment by krzkaczor** [comment]
@kkostalkowicz yeah I think it was a random glitch, I restarted CI to see if it helps. Side note: is gas estimation deterministic now? Or does it use some external data? If yes, then probably we need to hide it with `data-vis-reg-mask` or `data-vis-reg-hide` attributes.

File: cypress/utils/makeScreenshots.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ export function makeScreenshots(name: string, viewports: string[] = ['macbook-15
 
     for (const viewport of viewports) {
       cy.viewport(viewport as any);
+      cy.get('body').then(e => e.append('<textarea style="position: absolute; top: 0; opacity: 0" />'));
```
Comment (krzkaczor) [inline cypress/utils/makeScreenshots.ts]: i think we could move it out of the loop and do it only once. Also: what about removing it after? Maybe give it a unique ID and then it's easy to find it and remove.

**Pull request merged**